src/Modules/ModulesGraded.jl

```diff
@@ -543,7 +543,8 @@ end
 
 const AnyGradedRingElem = Union{<:MPolyDecRingElem, <:MPolyQuoRingElem{<:MPolyDecRingElem},
                                 <:MPolyLocRingElem{<:Ring, <:RingElem, <:MPolyDecRing},
-                                <:MPolyQuoLocRingElem{<:Ring, <:RingElem, <:MPolyDecRing}
+                                <:MPolyQuoLocRingElem{<:Ring, <:RingElem, <:MPolyDecRing}, 
+                                <:NCRingElement
```

Comment:
Now that I look at it again: Maybe we take out this line here. It was a custom fix to make my stuff on exterior algebras run.